Automattic/woocommerce-payments
# Fix scrollable content in currencies modal

Fixes #2145 

#### Changes proposed in this Pull Request

Fix scrollable area of "Add Currencies" Modal

The modal responsible for selecting active currencies for the multi-currency modal didn't have a maximum height, and when the currency count increased, it showed as a full height modal as described and pictured in #2145, also it had the title and search input also in the scroll content, this PR fixes it by wrapping the content which we want to scroll with a `div.add-enabled-currencies-modal__content`, and make it scrollable with `overflow-y: auto;` instead. 

Also this contains the fix for the mobile modal by stretching the list to fill the screen. The breakpoint chosen is `600px` to apply the fix, which is also the breakpoint when the modal turns to an overlay, but it can be changed.

#### Desktop Screenshot

![Screen Capture on 2021-06-22 at 00-41-35](https://user-images.githubusercontent.com/3295/122831662-e45ab300-d2f2-11eb-9435-f8702ef67bad.gif)


#### Mobile Screenshot

<img src='https://user-images.githubusercontent.com/3295/122831865-2b48a880-d2f3-11eb-827c-61c39a34a474.gif' width=300>


#### Testing instructions

* Navigate to WooCommerce > Settings
* Select the Multi-Currency tab
* Below the "Enabled Currencies" table, click the "Add currencies" button
* Desktop tests
	* Check the styling if the list height is bigger than 350px with more than 10-15 currencies visible (should be no)
	* Check the styling if the list height is smaller than 350px with less than 10-15 currencies visible (should be no, too)

* Mobile tests
	* Check if the buttons are on bottom, the header and search input is intact while the currency list is scrollable.

-------------------
